### PR TITLE
Add install on windows guide

### PR DESCRIPTION
**Pull request by ChangJoo-Park**
reference : https://elixir-lang.org/install.html#windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,18 @@ wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo
 `sudo apt-get install esl-erlang`
 + **Install Elixir**: `sudo apt-get install elixir`
 
+#### Windows:
+
++ Web installer
+  + [Download the installer](https://repo.hex.pm/elixir-websetup.exe)
+  + Click next, next, ..., finish
+
++ Chocolatey
```
Comment (iteles) [inline README.md]: Chocolatey?

Comment (ChangJoo-Park) [reply README.md]: https://chocolatey.org/ is package manager for Windows. this is commonly used by windows user

Comment (ChangJoo-Park) [reply README.md]: I think web installer and Chocolatey is same level

Comment (nelsonic) [reply README.md]: Would be good to get a clarification in-line fo others. 😉

**Pull request merged**